docs/tutorials/intro_tutorial.rst

```diff
@@ -10,22 +9,21 @@ Tutorial Description
 modeling <https://en.wikipedia.org/wiki/Agent-based_model>`__. Getting
 started with Mesa is easy. In this tutorial, we will walk through
 creating a simple model and progressively add functionality which will
-illustrate Mesa's core features.
+illustrate Mesa’s core features.
 
 **Note:** This tutorial is a work-in-progress. If you find any errors or
 bugs, or just find something unclear or confusing, `let us
 know <https://github.com/projectmesa/mesa/issues>`__!
 
 The base for this tutorial is a very simple model of agents exchanging
-money. Next, we add *space* to allow agents move. Then, we'll cover two
-of Mesa's analytic tools: the *data collector* and *batch runner*. After
-that, we'll add an *interactive visualization* which lets us watch the
+money. Next, we add *space* to allow agents move. Then, we’ll cover two
+of Mesa’s analytic tools: the *data collector* and *batch runner*. After
+that, we’ll add an *interactive visualization* which lets us watch the
 model as it runs. Finally, we go over how to write your own
 visualization module, for users who are comfortable with JavaScript.
 
 You can also find all the code this tutorial describes in the
-`examples/boltzmann_wealth_model/ <https://github.com/projectmesa/mesa/tree/master/examples/boltzmann_wealth_model>`__ directory of the Mesa
-repository.
+**examples/boltzmann_wealth_model** directory of the Mesa repository.
```

Comment:
Why remove the link?

Reply:
Must have done it accidentally...no issues keeping the link in